### ISSUE TITLE
native .query() hides properties in a join if there are columns with the same name in both tables 

### ISSUE THREAD

**Issue by jurajpetrik** [balderdashy/sails-postgresql#173]
I want to run a native query using sails-postgresql with a join of two tables. Both of these tables contain a column that's named the same. In the example below both pet and owner have a column name. When I run the query in postgresql console it displays it fine. However if I run the same query using Model.query(), one of the columns gets overwritten.

     $ select * from pet;
      id |   name   
     ----+----------
      1  | some pet
     (1 row)
     
     $ select * from owner;
      id |    name    | petid 
     ----+------------+-------
      1  | some owner | 1
     (1 row)
     
     $ select * from pet JOIN owner on pet.id=owner.petId;

      id |   name   | id |    name    | petid 
     ----+----------+----+------------+-------
      1  | some pet | 1  | some owner | 1
   
same query as above but run using sails-postgresql

     Pet.query('select * from pet JOIN owner on pet.id=owner.petId;').exec((err,res)=>{
     console.log(res.rows);
     }

    //Missing columns with the same name

    //[{id:1,
       name:some owner,
       petid:1}]

**Comment by kevinburkeshyp** [comment]
Not sure there is a great solution here, I am fairly certain the `pg` library under this one exhibits the same behavior. I worked around this by using `AS` to dedupe the column rows... every nonduplicate row you can still group under `pet.*`

**Comment by sailsbot** [comment]
Thanks for posting, @jurajpetrik.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
